src/app-insight.service.ts

```diff
@@ -6,7 +6,7 @@ import { filter } from 'rxjs/operators';
 import IAppInsights = Microsoft.ApplicationInsights.IAppInsights;
 
 export class AppInsightsConfig implements Microsoft.ApplicationInsights.IConfig {
-  instrumentationKeySetlater?: boolean;
+  instrumentationKeySetLater?: boolean;
```

Comment:
Changing this will be a breaking change unless you keep the old property and proxy it to this one.

My recommendation is to just live with the typo

Reply:
Or live on my fork.
Of course i can add new field with typo fix and rewrite code to use new and old field for this PR. How you look at it?

Reply:
Would like to get @MarkPieszak 's opinion on this. Maybe he can include this if we bump a major version.